src/compareNums.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+'use strict';
+
+const compareNumbers = (input, randomNum) => {
+  let bulls = 0;
+  let cows = 0;
```

Comment:
```suggestion
  const result = {
    bulls: 0,
    cows: 0,
  };
```

Return an object from this function with comparison result